x/datapool/types/pool.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/v044_temp/address"
+)
+
+const (
+	PENDING = "PENDING"
+	ACTIVE  = "ACTIVE"
+)
+
+func NewPool(poolId uint64, curator sdk.AccAddress, poolParams PoolParams) *Pool {
```

Comment:
```suggestion
func NewPool(poolID uint64, curator sdk.AccAddress, poolParams PoolParams) *Pool {
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Likewise, I think NFT would be better than Nft.

Reply:
modified in [c2f5cf9](https://github.com/medibloc/panacea-core/pull/287/commits/c2f5cf906992109ff1cfd74749ebe2b0e6782b70) and [f953e5c](https://github.com/medibloc/panacea-core/pull/287/commits/f953e5ca1499bdebe492b75185bd47a369c82032)